index.rst

```diff
@@ -437,284 +329,111 @@ this case, we may have something like:
+We propose that, while in ``STANDBY`` state, CSCs continuously monitor the configuration repository and update the information if needed.
+If an invalid configuration is made available while the CSC is in ``STANDBY`` the CSC should transition to ``FAULT`` state.
+
+.. _section-default-configuration:
+
+Base configuration (handling default configuration values)
+----------------------------------------------------------
+
+This is mainly a proposal to update how Salobj manages default configuration values.
+Other :ref:`handcrafted CSCs <section-handcrafted>` are encouraged to follow this proposal as closely as possible to maintain uniformity across the system.
 
-    #.  Rename ``settingsVersions``to``configurationsAvailable``
+As described :ref:`above <section-salobj>`, CSCs written with Salobj define a configuration schema (e.g. `ts_atdome <https://github.com/lsst-ts/ts_ATDome/blob/develop/schema/ATDome.yaml>`__).
+The configuration schema contains default values for the configuration which are loaded if the ``start`` command is sent with an empty ``configuration`` attribute (the default value).
+Nevertheless, the values in the schema are seldom valid beyond a unit testing environment, which requires users to provide some kind of *operational defaults* or *default label*.
+At the same time, overriding only a small subset of the *schema defaults* is usually enough for operations.
+Therefore, to get a full set of applied configurations, users must look at two distinct repositories; the configuration repository (for the modified parameters) and the CSC repository (for the schema defaults).
 
-        #.  Rename ``recommendedSettingsLabels`` to ``labels``.
-        #.  Rename ``recommendedSettingsVersion`` to ``version`` or
-            ``branchDescription``.
-        #.  Rename ``settingsUrl`` to ``url``.
-        #.  Add ``mapping`` or ``filename``. Filename might not be suitable for
-            database based configuration.
+The proposal to improve this aspect of the system is:
 
-    #. Rename ``settingsApplied``to ``configurationApplied``
+#.  Remove default values from the configuration schema.
+#.  On the configuration repository there must be a ``_base.yaml`` file defining all the base configuration values (we use "base" instead of "default").
+#.  The labels file (e.g. ``_labels.yaml``) will continue to exist with the same format and purpose.
+#.  Additional configuration files can provide new values for individual configuration parameters.
+#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it will load the values in ``base.yaml``.
+#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it will load the values in ``base.yaml`` first and override those values defined in the mapped configuration file.
```

Comment:
We need something in the new config file that states explicitly that the values of base.yaml are to be loaded and the config file only overwrites the values. 
We should really build a configuration file template!! in fact there is a ticket for this in the docs jira project 
https://jira.lsstcorp.org/browse/PREOPS-27